en/collect/jabref-browser-extension.md

```diff
@@ -26,7 +26,7 @@ Most JabRef installations include the necessary files, so test the extension bef
 2. Install the JabRef browser extension: [Firefox](https://addons.mozilla.org/en-US/firefox/addon/jabref/?src=external-github), [Chrome](https://chrome.google.com/webstore/detail/jabref-browser-extension/bifehkofibaamoeaopjglfkddgkijdlh), [Vivaldi](https://chrome.google.com/webstore/detail/jabref-browser-extension/bifehkofibaamoeaopjglfkddgkijdlh)
 3. On Windows 7, please [upgrade Powershell](https://www.microsoft.com/en-us/download/details.aspx?id=54616).
 4. Download the following files and copy them to the same directory as `JabRef.exe`
-   * [jabref.json](https://raw.githubusercontent.com/JabRef/jabref/master/buildres/windows/jabref.json)
+   * [jabref-firefox.json](https://raw.githubusercontent.com/JabRef/jabref/master/buildres/windows/jabref-firefox.json)
```

Comment:
This is 404 - does some PR need to be merged?

Reply:
Yes, that change is in https://github.com/JabRef/jabref/pull/7056

Reply:
@LyzardKing: Even after merging of JabRef/jabref#7056, the deadlink remains dead.

Reply:
You're right. I changed the filename in https://github.com/JabRef/jabref/pull/7092

Reply:
Great!